source/manual/access-to-licensing-for-third-parties.html.md

```diff
@@ -0,0 +1,58 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Access to Licensify for Third Parties
+parent: "/manual.html"
+layout: manual_layout
+section: 2nd line
+type: learn
+---
+
+[Licensify](/manual/licensing.html) is a GOV.UK application which is usually supported by third parties.
+
+This document explains how to access the Licensify infrastructure as a third party, to perform various maintenance tasks.
+
+## Accessing the logs
+
+Licensify uses an Elasticsearch / Logstash / Kibana system hosted by Logit.io for its logs.
+
+If you haven't already got access to GDS' Logit account, you'll need to ask a GOV.UK developer to set you up.
+
+Once you have an account, you can [follow these instructions to sign in](https://reliability-engineering.cloudapps.digital/logging.html#content).
+
+## Accessing the VPN
+
+This is a prerequisite for accessing Jenkins (which is used for deployments), and for SSH-ing onto instances.
+
+Follow [the VPN guidance for non-GDS devices ("BYOD")](https://docs.google.com/document/d/150JX1xiWdXY29ahcYUMb05Si-hEAZvtkGAKojT9Rjis/edit)
+to set up the VPN. You will need to sign into your `@digital.cabinet-office.gov.uk` Google account to access this document.
+
+## Deploying code with Jenkins
+
+Licensify is built and deployed using Jenkins.
+
+The process for building a new release and promoting it through the three
+environments is as follows:
+
+1. When a PR/branch is merged into master, this kicks a new build of master on
+   [the CI Jenkins](https://ci.integration.publishing.service.gov.uk/job/licensify/). This
+   produces the necessary artefacts to deploy Licensify. Each build is given a build number.
+1. The new master build will be automatically be deployed to integration via
```

Comment:
Recommend changing "via" to "using" as "via" is not within style as not plain English